### PR TITLE
clean code

### PR DESCRIPTION
**Pull request by oneWayOut**
change a little bit to make the code more readable

### CODE REVIEW

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp [hunk 1]
```diff
@@ -829,16 +829,6 @@ FixedwingPositionControl::control_position(const math::Vector<2> &curr_pos, cons
 				wp_distance_save = 0.0f;
 			}
 
-			// create virtual waypoint which is on the desired flight path but
```
Comment (TSC21) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: why is this removed?

File: src/modules/fw_att_control/fw_att_control_main.cpp [hunk 2]
```diff
@@ -1041,6 +1030,11 @@ FixedwingAttitudeControl::task_main()
 					_wheel_ctrl.reset_integrator();
 				}
 
+				float roll_sp = _att_sp.roll_body;
```
Comment (TSC21) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: Maybe add a line comment saying `/* Update atittude setpoint variables */`

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp (hunk 1 above)
Comment (oneWayOut) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: Because it's not used anywhere.

File: src/modules/fw_att_control/fw_att_control_main.cpp (hunk 2 above)
Comment (oneWayOut) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: As there is no comment before, I just leave it as that.  Maybe you can add as you wish.

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp (hunk 1 above)
Comment (TSC21) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: Did you flight test with these changes?

Comment (dagar) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: Thanks @oneWayOut. I was actually thinking of restoring this functionality, but we might as well delete this unused chunk until I get around to it.

File: src/modules/fw_att_control/fw_att_control_main.cpp (hunk 2 above)
Comment (TSC21) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: It was you that added these variables before at this point so if you don't mind adding this comment, we appreciate.

**Comment by dagar** [comment]
Looks good, I'll merge once CI passes. There's still a lot of dead code in these controllers that make them seem more complicated than they actually are.

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp (hunk 1 above)
Comment (oneWayOut) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: In SITL, the new code works well. I have not do a real flight test yet. @TSC21

File: src/modules/fw_att_control/fw_att_control_main.cpp (hunk 2 above)
Comment (oneWayOut) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: do I need to do a new PR? @TSC21

Comment (TSC21) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: A new commit. But let it as is, no issue if there's no description.

File: src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp (hunk 1 above)
Comment (dagar) [inline src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp]: That's definitely good practice, although in this case I think SITL is sufficient.
